app/screens/select_server/select_server.js

```diff
@@ -205,20 +208,22 @@ class SelectServer extends PureComponent {
                                 defaultMessage='Enter Server URL'
                             />
                         </View>
-                        <TextInputWithLocalizedPlaceholder
-                            ref={this.inputRef}
-                            value={serverUrl}
-                            editable={allowOtherServers}
-                            onChangeText={this.props.actions.handleServerUrlChanged}
-                            onSubmitEditing={this.onClick}
-                            style={[GlobalStyles.inputBox, allowOtherServers ? {} : {backgroundColor: '#e3e3e3'}]}
-                            autoCapitalize='none'
-                            autoCorrect={false}
-                            keyboardType='url'
-                            placeholder={{id: 'mobile.components.select_server_view.siteUrlPlaceholder', defaultMessage: 'https://mattermost.example.com'}}
-                            returnKeyType='go'
-                            underlineColorAndroid='transparent'
-                        />
+                        {!Config.AutoSelectServerUrl &&
```

Comment:
How does this render if AutoSelectServerUrl is set? It is just "Enter Server URL" with a "Proceed" button below it?

Reply:
ah yeah, the "Enter Server URL" should be hidden as well. I'll make that change.